dotCMS/src/main/java/com/dotmarketing/util/ImportUtil.java

```diff
@@ -1517,7 +1520,7 @@ private static void importLine ( String[] line, String currentHostId, Structure
     }
 
     private static Contentlet runWorkflowIfCould(final User user, final List<Permission> contentTypePermissions,
-                                                 final Set<Category> categories, final Contentlet contentlet,
+                                                 final List<Category> categories, final Contentlet contentlet,
```

Comment:
why changing from Set to List?

Reply:
the list of categories already comes ready to process, in that method we're only converting to a list when passing to the following methods, so it's better to receive a list.

Reply:
But the Set avoids duplicates. Is that something we want to keep?

Reply:
I know, but as I said the list comes ready to this method, meaning without duplicates. We're not adding more categories inside this method, so there is no really an advantage to use Set, since the other methods are waiting for a list.